tools/autograd/gen_python_functions.py

```diff
@@ -165,6 +191,7 @@ def get_py_torch_functions(declarations):
     def should_bind(declaration):
         return (should_generate_python_binding(declaration) and
                 not is_nn_module_function(declaration) and
+                not is_fft_module_function(declaration) and
```

Comment:
err...should we change this around so it's no namespace?  (Might need to leave nn module because of THNN stuff)

Reply:
Let's leave this for the torch.linalg namespace PR?